torch/nn/_functions/rnn.py

```diff
@@ -309,8 +310,68 @@ def forward(input, *fargs, **fkwargs):
         import torch
         if torch._C._jit_is_tracing(input):
             import torch.onnx.symbolic
+            sym = torch.onnx.symbolic.RNN_symbolic_builder(*args, **kwargs)
+            cell_type = args[0]
+
+            def rnn_trace_override_symbolic(g, input, weights, hiddens, batch_sizes):
```

Comment:
I guess it's ok to have 10 line symbolics in different files, but inlining this into `RNN` is too much. Can this be moved to `torch.onnx` somehow? It basically reimplements the RNN which is not great.